model/ids.go

```diff
@@ -77,6 +77,21 @@ func TraceIDFromString(s string) (TraceID, error) {
 	return TraceID{High: hi, Low: lo}, nil
 }
 
+// TraceIDFromBytes creates a TraceID from list of bytes
+func TraceIDFromBytes(data []byte) (TraceID, error) {
+	var t TraceID
+	switch {
+	case len(data) == traceIDLongBytesLen:
+		t.High = binary.BigEndian.Uint64(data[:traceIDShortBytesLen])
+		t.Low = binary.BigEndian.Uint64(data[traceIDShortBytesLen:])
+	case len(data) == traceIDShortBytesLen:
+		t.Low = binary.BigEndian.Uint64(data)
+	default:
+		return TraceID{}, fmt.Errorf("invalid length for TraceID")
+	}
+	return t, nil
+}
```

Comment:
Since we're adding a new function, I would prefer to reuse it from `Unmarshal(data []byte)`, e.g.
```
func (t *TraceID) Unmarshal(data []byte) error {
    var err error
    *t, err = TraceIDFromBytes(data)
    return err
}
```

Similar for Unmarshal on SpanID

Reply:
Done. And this change would let `func Unmarshal` parse 8byte long TraceID(Higher 8bytes will be 0)